specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get server metrics configuration by a test or test run": {
+            "$ref": "./examples/GetServerMetricsByTestOrTestRun.json"
+          }
+        }
+      }
+    },
+    "/serverMetricsConfig/default": {
+      "get": {
+        "tags": [
+          "ServerMetrics"
+        ],
+        "summary": "Get all default server metrics configuration for supported resource types.",
+        "operationId": "ServerMetrics_GetDefaultMetrics",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The requested default server metrics",
+            "schema": {
+              "$ref": "#/definitions/DefaultServerMetricsConfigListModel"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get default server metrics configuration": {
+            "$ref": "./examples/GetServerDefaultMetrics.json"
+          }
+        }
+      }
+    },
+    "/serverMetricsConfig/supportedResourceTypes": {
+      "get": {
+        "tags": [
+          "ServerMetrics"
+        ],
+        "summary": "Get all supported resource types for adding an app component(Azure resource types).",
+        "operationId": "ServerMetrics_ListSupportedResourceType",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The supported resource types and metrics for server side metrics",
+            "schema": {
+              "$ref": "#/definitions/SupportedResourceType"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get list of supported resource types for configuring server metrics": {
+            "$ref": "./examples/ListSupportedResourceType.json"
+          }
+        }
+      }
+    },
+    "/loadtests/{testId}": {
```

Comment:
Recommend changing this path to simply "/tests". This will be consistent with 1) the response model name (after it is fixed to remove "model") 2) the external docs, and 3) the "/testruns" path, which is not "/loadtestruns".

Reply:
changed